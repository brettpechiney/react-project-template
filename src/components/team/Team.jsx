import React from 'react';
import { hot } from 'react-hot-loader';
import './team.scss';

const Team = () => (
  <section className="team--container">
    <p className="team-intro">This is the team page.</p>
  </section>
);

export default hot(module)(Team);
