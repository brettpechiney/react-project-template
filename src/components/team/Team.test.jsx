import React from 'react';
import renderer from 'react-test-renderer';

import Team from './Team';

describe('<Team />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Team />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
