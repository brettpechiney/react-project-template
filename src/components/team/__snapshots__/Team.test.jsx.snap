// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Team /> renders correctly 1`] = `
<section
  className="src-components-team-___team__team___1IUwF"
>
  <p
    className="src-components-team-___team__intro___KGWa0"
  >
    This is the team page.
  </p>
</section>
`;
